cloud_config_workaround.sh

```diff
@@ -69,9 +69,13 @@ case ${SteamAppId} in
         CONFIG_PATH="Documents/My Games/CRISIS CORE FINAL FANTASY VII REUNION/Steam/${STEAMID}"
         CONFIG="SAVEDATA_SYSTEM.sav"
         ;;
-    2337640)
-        CONFIG_PATH="AppData/Local/PinballM/Saved/SaveGames/${STEAMID}"
-        CONFIG="settings.sav"
```

Comment:
@PrototypeActual Can you add this back in? Then I'll merge it

Reply:
Ah, I thought this was still included. I'm remote this week but I'll see if I can add this in within the week